src/test/regress/upgrade/utils.py

```diff
@@ -0,0 +1,31 @@
+import subprocess
+import os
+from config import USER
+
+
+def run(command):
+    return subprocess.call(command)
```

Comment:
I don't see a reason to have this a utility, we can just use `subrpocess.call` instead of `utils.run`

Reply:
initially I was passing `shell=True`, after removing it, it doesnt really have any utility as we are passing only one argument without any additional things so I will remove this method and use `subprocess.call` directly